recipes/vulkan-loader/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.3.231":
+    url: "https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.231.tar.gz"
+    sha256: "02e185b939635167ea8f8815f8daab76af36923a3b995951fe6a5d3e25c55bf7"
```

Comment:
Maybe wait for the sdk flavor of 1.3.231? It avoids to maintain too many versions.

Reply:
The last version was about 2 months ago (the last one before that was another 4 months prior), not terrible but I had reason to update:

1. I already made the corresponding vulkan-headers version (1.2.231). It would be a bit weird to have vulkan-loader missing.
2. I think that a version where there's some dependency change seems like a good point to update the recipe

If it becomes an issue, it could be removed (package stays but recipe not updated), maybe long term maybe have a policy where only SDK releases are treated as "LTS"?